### ISSUE TITLE
Information Architecture modification -> Vision Accessibility audit

### ISSUE THREAD

**Issue by develric** [wordpress-mobile/WordPress-Android#10962]
We need to Audit all the modifications introduced with IA project to ensure we correctly support Vision Accessibility.

Reporting below some note by @shiki as a starting point:

> - Reader: When the filter control is selected, TalkBack just says the filter name (e.g. “All sites”). It doesn’t say that it’s a button or that it’s a filter. It’s not obvious what happens after tapping, basically. It might be better for TalkBack to say something like “Filter, , Button”.
> - Reader: The Settings button is announced as “Unlabeled, Button”.
> - Reader → Filter List: The Sites and Tags are not marked as headings. A heading attribute is useful if you’re [navigating by heading ](https://href.li/?https://support.google.com/accessibility/android/answer/6006598?hl=en)with TalkBack.
> - Reader → Filter List: The currently selected filter is not announced by TalkBack as “selected”.
> - My Site: The “Stats”, “Pages”, “Posts”, and “Media” buttons are spoken as “Unlabeled button”.
> - My Site: The “Stats”, “Pages”, “Posts”, and “Media” labels are unnecessarily accessible. Since the buttons (icons) are accessible, these don’t need to be.
